packages/stack-shared/src/config/db-sync-mappings.ts

```diff
@@ -0,0 +1,166 @@
+export const DEFAULT_DB_SYNC_MAPPINGS = {
+  "PartialUsers": {
+    sourceTables: { "ContactChannel": "ContactChannel", "ProjectUser": "ProjectUser" },
+    targetTable: "PartialUsers",
+    targetTablePrimaryKey: ["id"],
+    targetTableSchema: `
+   CREATE TABLE IF NOT EXISTS "PartialUsers" (
+        "id" uuid PRIMARY KEY,
+        "createdAt" timestamp with time zone,
+        "updatedAt" timestamp with time zone,
+        "type" text,
+        "isPrimary" boolean,
+        "isVerified" boolean,
+        "value" text,
+        "sequenceId" bigint,
+        "userUpdatedAt" timestamp with time zone,
+        "profileImageUrl" text,
+        "displayName" text,
+        "userCreatedAt" timestamp with time zone,
+        "isAnonymous" boolean
+      );
+      CREATE INDEX ON "PartialUsers" ("sequenceId");
+      REVOKE ALL ON "PartialUsers" FROM PUBLIC;
+      GRANT SELECT ON "PartialUsers" TO PUBLIC;
+    `.trim(),
```

Comment:
can we replace this by just having some code in the externalDbUpdateQuery which does something along the lines of "if table does not exist yet, create it"? that seems like a safer and cleaner way to initialize the schema (eg. imagine what happens when a user wipes the database)